### PR TITLE
fix: add validation to role selectDropdown

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/iam/components/UserManagementTable.vue [hunk 1]
```diff
@@ -238,6 +238,11 @@ const handleRemoveButton = async () => {
         modalState.loading = false;
     }
 };
+
+const isWorkspaceGroupUser = (item: ExtendUserListItemType) => {
+    console.log(item);
```
Comment (yuda110) [inline apps/web/src/services/iam/components/UserManagementTable.vue:243]: nitpick: here!

File: apps/web/src/services/iam/components/UserManagementTable.vue [hunk 2]
```diff
@@ -302,11 +298,20 @@ const handleRemoveButton = async () => {
...
                                        class="role-select-dropdown"
                                        @select="handleSelectDropdownItem($event, rowIndex)"
                     >
                         <template #dropdown-button>
+                            <p-tooltip position="bottom"
+                                       :contents="useRoleFormatter(value?.type).name"
+                                       class="tooltip"
+                            >
+                                <img :src="useRoleFormatter(value?.type).image"
+                                     alt="role-type-icon"
```
Comment (yuda110) [inline apps/web/src/services/iam/components/UserManagementTable.vue:311]: quibble: The `alt` attribute should contain easily readable text, something like "Role Type Icon"!

**Review by yuda110 — APPROVED** [review]
LGTM!!
I left a comment, but you can ignore it😅

File: apps/web/src/services/iam/components/UserManagementTable.vue (hunk 2 above)
Comment (sulmoJ) [reply apps/web/src/services/iam/components/UserManagementTable.vue:311]: GOOOOD, Thx U

**Pull request merged**